ST_mortality.c

```diff
@@ -692,31 +776,37 @@ static void _age_independent( const SppIndex sp) {
 
 }
 
-/***********************************************************/
+/**
+ * \brief Kills individuals to simulate death by insufficient resources.
+ * 
+ * Note that this function does NOT check if resource availibility is low.
+ * It kills an amount of individuals proportional to resource limitations,
+ * but deciding if any individuals should be killed at all (whether or
+ * not to call this function) takes place in mort_Main().
+ * 
+ * In reality, resource death would occur gradually over the growing
+ * season. However, in this yearly-time-step model, it must be done either
+ * before or after the growing season. 
+ * 
+ * This function takes an additional step with clonal species. It calls
+ * _stretched_clonal() to kill additional amounts of clonal plants due 
+ * to insufficient resources.
+ * 
+ * Initial programming by Chris Bennett @ LTER-CSU 6/15/2000.
+ * 
+ * \param rg The Index in RGroup of the group to kill.
+ * 
+ * \sideeffect If this function decided to kill any individuals, which
+ *             it most likely will, the individual linked lists of every
+ *             species in RGroup[rg] will be modified.\n
```

Comment:
same comment as above - strangely stated

Reply:
@kpalmqui I modified the comment in commit 984bd13. Let me know what you think!